x/stakeibc/keeper/unbonding.go

```diff
@@ -206,7 +202,13 @@ func SortUnbondingCapacityByPriority(validatorUnbondCapacity []ValidatorUnbondCa
 		validatorA := validatorUnbondCapacity[i]
 		validatorB := validatorUnbondCapacity[j]
 
-		// TODO: Once more than 32 validators are supported, change back to using balance ratio first
+		balanceRatioValA, _ := validatorA.GetBalanceRatio()
+		balanceRatioValB, _ := validatorB.GetBalanceRatio()
+
+		// Sort by the balance ratio first - in ascending order - so the more unbalanced validators appear first
```

Comment:
If the default value is literally 100 and we can reach all validators now, why is sorting at all important?  There is no cutoff for the order here to matter right?

Reply:
disregard this change - I've since reverted it

But to answer your questions, it's important in the case that we have to fall back down to 32 validators, and it makes the unbondings more "fair" instead of just targeting the top vals